crates/iota-indexer/src/types.rs

```diff
@@ -416,6 +416,27 @@ impl EventIndex {
     }
 }
 
+#[cfg(feature = "pg_integration")]
```

Comment:
I would suggest exposing this for all tests:

```suggestion
#[cfg(test)]
```

Reply:
fixed: https://github.com/iotaledger/iota/pull/7969/commits/5f601e42b9603eab8de7de10f29c082f5f622b75